temporalcli/commands.workflow_test.go

```diff
@@ -261,26 +262,41 @@ func (s *SharedServerSuite) TestWorkflow_Terminate_SingleWorkflowSuccess_WithRea
 }
 
 func (s *SharedServerSuite) TestWorkflow_Terminate_BatchWorkflowSuccess() {
-	res := s.testTerminateBatchWorkflow(false)
+	_, res := s.testTerminateBatchWorkflow(5, 0, false)
 	s.Contains(res.Stdout.String(), "approximately 5 workflow(s)")
 	s.Contains(res.Stdout.String(), "Started batch")
 }
 
-func (s *SharedServerSuite) TestWorkflow_Terminate_BatchWorkflowSuccessJSON() {
-	res := s.testTerminateBatchWorkflow(true)
+func (s *SharedServerSuite) TestWorkflow_Terminate_BatchWorkflow_Ratelimit_MissingQuery() {
+	res := s.Execute(
+		"workflow", "terminate",
+		"--address", s.Address(),
+		"--rps", "5",
+	)
+	s.Error(res.Err, "must set either workflow ID or query")
+}
+
+func (s *SharedServerSuite) TestWorkflow_Terminate_BatchWorkflowSuccess_Ratelimit() {
+	events, _ := s.testTerminateBatchWorkflow(2, 1, false)
+	delay := events[1].EventTime.AsTime().Sub(events[0].EventTime.AsTime()).Abs()
+	s.InDelta(delay, 1*time.Second, float64(100*time.Millisecond))
```

Comment:
In my tests it hasn't been exactly 1s, so I added some delta here.

Reply:
This is the kind of test that grows flaky with Go runner slowness I've found. Expecting this margin of error is a bit rough. I think for this you may only need to test that it's set on the outbound request (e.g. w/ gRPC interceptor maybe), you don't need to test server behavior (that's for the server to test).